.github/workflows/rust.yml

```diff
@@ -40,6 +40,10 @@ jobs:
       env:
         LLVM_PROFILE_FILE: coverage/serialize-escape-html-%p-%m.profraw
       run: cargo test --features serialize,escape-html
+    - name: Run tests (all features)
+      env:
+        LLVM_PROFILE_FILE: coverage/all-features-%p-%m.profraw
+      run: cargo test --all-features
```

Comment:
Looks like this should cover testing both with and without `overlapped-lists` which is a concern because of so much alternate code :+1:

Reply:
Yes, that is why I add this.